geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/domain/DataCommandResult.java

```diff
@@ -381,6 +397,12 @@ public ResultModel toResultModel() {
     }
 
     ResultModel result = new ResultModel();
+
+    if (warningMessage != null && !warningMessage.isEmpty()) {
+      InfoResultModel info = result.addInfo(HEADER_INFO_SECTION);
```

Comment:
It might be more consistent to either call `setHeader` here instead of `addInfo`, or to rename `HEADER_INFO_SECTION` to `WARNING_INFO_SECTION` or something similar.

Reply:
I'm reluctant to change the addInfo to setHeader because I want the structure of the output to match remove as closely as possible so I opted to change the name to WARNING_INFO_SECTION as this section isn't used in any other context.